features/core-install.feature

```diff
@@ -171,6 +171,35 @@ Feature: Install WordPress core
       """
     And the return code should be 0
 
+  Scenario: Install WordPress with locale set to de_DE
```

Comment:
Can you add another test with an older version of Core to make sure that it shows a warning (as described below) and still correctly installs the Core (in `en_US`)?